src/components/Tabs/Tabs.tsx

```diff
@@ -1 +1,43 @@
-export const Tabs = () => {};
+import { Tab } from '../../types/Tab';
+import { TabInfo } from '../Tab/TabInfo';
+
+type Props = {
+  tabs: Tab[],
+  selectedTabId: string,
+  onTabSelected: (tab: Tab) => void;
+};
+
+export const Tabs = ({ tabs, selectedTabId, onTabSelected }:Props) => {
+  const selectedTabIdAdjusted = tabs
+    .map(tab => tab.id)
+    .includes(selectedTabId) ? selectedTabId : tabs[0].id;
```

Comment:
this i absolutely fine, but using `.find()` method would be a bit simpler and more readable

Reply:
Totally agreed